riscv/riscv.s

```diff
@@ -10,10 +10,11 @@
 # https://github.com/jameslzhu/riscv-card/blob/master/riscv-card.pdf
 
 ## Saving numbers to registers
-# There's no way to directly store a value in a register
+# There's no way to directly store a value in a register 
```

Comment:
Totally nbd but seeing the extra space here, I wanted to mention that one setting I turned on my text editor is to trim whitespace on save- it's one of my favorite things!!

Reply:
sorry about this. ooo thank you! just downloaded an extension for this :))